willow/plugins/svglib.py

```diff
@@ -0,0 +1,30 @@
+from io import BytesIO
+
+from svglib.svglib import svg2rlg
```

Comment:
We should move these imports somewhere where they wouldn't be run unless an operation is called. ``svglib`` and ``reporlab`` are both optional dependencies and this file is imported form ``willow/__init__.py``

Reply:
Updated in https://github.com/wagtail/Willow/pull/99/commits/4a030171693eae2111ad3d10d7a57edf225a326c